### PR TITLE
D8CORE-850 Ckeditor Sticky Toolbar

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Enables sticky toolbar module

# Need Review By (Date)
- 12/11

# Urgency
- low

# Steps to Test
1. `compose require su-sws/stanford_profile:dev-D8CORE-850-ckeditor-sticky --prefer-source`
1. `drush cim -y` or `blt drupal:install`
1. create a page with a wysiwyg
1. add a ton of text
1. scroll down and verify the toolbar sticks to the top of the window.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
         "drupal/better_normalizers": "^1.0-beta3",
         "drupal/block_blacklist": "~1.0",
         "drupal/chosen": "~2.4",
+        "drupal/ckeditor_fixed_toolbar": "dev-1.x#1e4a8191ab289b4c91d9a0f2ac6b40e82e294908",
```
Comment (sherakama) [inline composer.json]: No stable releases :(
Not much movement in the last 12 months :(

I had a read through the module and it doesn't do much so this dev version is likely ok and it won't be the end of the world if we have to remove it later due to a deprecation or something. 

@boznik Can you have a review of this module (https://git.drupalcode.org/project/ckeditor_fixed_toolbar) and let me know if there is anything concerning in there for you.

Comment (pookmish) [reply composer.json]: yeah its really just a dozen lines of code. super slim

**Comment by sherakama** [comment]
@boznik

**Comment by sherakama** [comment]
Yo @boznik!

**Comment by pookmish** [comment]
poke

**Pull request merged**